docs/user/device_owner.rst

```diff
@@ -0,0 +1,45 @@
+Device Owner
+============
+
+Create Device Owner Account
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to manage Kolibri content channels, data and users, you must first create a **Device Owner** account and the **Facility**. The **Device Owner** account registration page appears when you start the server for the first time  after the installation of Kolibri and open the browser at http://127.0.0.1:8080/.
```

Comment:
If it is changeable, maybe it would be good to say something along the lines of "e.g. 127.0.0.1:8080", or mention that it is by default ":8080"?